tests/test_generator.py

```diff
@@ -1,23 +1,45 @@
 """Test JSON schema fuzzer."""
 import glob
-import json
 import re
 from pathlib import Path
 
+import jsonpickle
 import jsonschema
 import pytest
 
 from json_schema_fuzz import generate_json
+from json_schema_fuzz.utils import custom_json_loads
+
+# Create a custom validator
+# for our custom properties
+
+
+def not_multiple_of_validator(validator, value, instance, schema):
+    """ jsonschema validator function for custom notMultipleOf property """
+    if not isinstance(value, list):
+        value = [value]
+    for num in value:
+        if instance % num == 0:
```

Comment:
The way most JSON Schema keywords are defined, they can only invalidate an instance if the instance is of the appropriate type. In this case, I think `notMultipleOf` should validate if the instance is non-numeric. We should check the type here.